assets-src/styles/sass/50-core-components/_navigation.scss

```diff
@@ -7,24 +7,18 @@
 }
 
 #global-nav {
-	border-bottom: solid 1px $border-color;
 	display: block;
-	padding-bottom: rem(18);
-	padding-top: rem(18);
-
-	@include mq($max-width) {
-		padding-bottom: 0;
-		padding-top: rem(20);
-	}
+	padding-bottom: 1.6em;
+	padding-top: 1.6em;
```

Comment:
@ostafinskim it would make more sense to use the `em` mixin for this so it is consistent with the rest of the CSS. 

1.6 is also an awkard pixel value (1.6 * 16 = 25.6 pixels. If you want to use em here then 1.625 would be better as 1.625 * 16 = 26 pixels).